webapp/src/components/StartingMaterialInformation.vue

```diff
@@ -84,9 +79,13 @@
 import { createComputedSetterForItemField } from "@/field_utils.js";
 import TinyMceInline from "@/components/TinyMceInline";
 import ChemicalFormula from "@/components/ChemicalFormula";
+import ChemFormulaInput from "@/components/ChemFormulaInput";
 import TableOfContents from "@/components/TableOfContents";
-import CollectionList from "@/components/CollectionList";
+import ToggleableCollectionFormGroup from "@/components/ToggleableCollectionFormGroup";
 import FormattedRefcode from "@/components/FormattedRefcode";
+import StyledInput from "@/components/StyledInput";
+
+import { IS_STARTING_MATERIAL_EDITABLE } from "@/resources.js";
```

Comment:
Must say that I don't really like this; would prefer it to be added as config on the server side that the app can query in the style of #455 (eventually I'd like all "app" config except the API URL to be made server metadata).

Does it really need to be configurable? The other option would be that admins can always do it, and you only show the button if they are an admin...

Reply:
I'll make it configurable from an environment variable now, since that's how all the other webapp config is done. A clean way of storing the specifying the app config is definitely something we should.

As for why this needs to be configurable: groups that are importing their inventory from a 3rd party chemical inventory manager need it to be "false" in most cases, to avoid the two inventories getting out of sync. Groups that want to use datalab as their chemical inventory will need it to be "true". If we add more roles like "inventory manager" in the future, it will be easy to fine-grain who can edit these fields.

Reply:
Ok, now its an environment variable with default "false". Name has been changed to "EDITABLE_INVENTORY"